src/app/results/results.component.ts

```diff
@@ -76,7 +76,7 @@ export class ResultsComponent implements OnInit {
 
   filterByDate() {
     let urldata = Object.assign({}, this.searchdata);
-    urldata.query += urldata.query.substr(urldata.query.length - 6, 6) !== " /date" ? " /date" : "";
+    // urldata.query += urldata.query.substr(urldata.query.length - 6, 6) !== " /date" ? " /date" : "";
```

Comment:
No need to comment this part, since the method call is already been commented out in the component.html file. This might create ambiguity in understanding code afterwards. Please remove it and rest is all fine :+1: